main/unbound/APKBUILD

```diff
@@ -69,10 +67,9 @@ package() {
 
 	cd "$pkgdir"
 
-	install -Dm755 "$srcdir"/update-unbound-root-hints \
-		./etc/periodic/monthly/update-unbound-root-hints
-
-	install -m644 -D "$srcdir"/root.hints ./etc/unbound/root.hints
+	mkdir -p ./etc/unbound
+	rm -f ./etc/unbound/root.hints
+	ln -s ../../usr/share/dns-root-hints/named.root ./etc/unbound/root.hints
```

Comment:
Why not pointing to an absolute path?

Reply:
An absolute path only works when the target exists in exactly one location. This way the link works in chroot or under $srcdir too.